### DISCUSSION TITLE
Desingn of Checklists

### DISCUSSION THREAD

**Discussion by Shadow-of-eagle** [FossifyOrg/General-Discussion#610] [General]
I just wanted to say, that I prefer the design of checklists with the crosses and ticks like it was in Simple Notes (first image) over the design with the checkboxes like it is in Fossify Notes (second image).
<img alt="Checklist in Simple Notes" src="https://github.com/FossifyOrg/Notes/assets/128985883/2487b506-9ba7-486a-8551-2045e337c8af" width=200 />
<img alt="Checklist in Fossify Notes" src="https://github.com/FossifyOrg/Notes/assets/128985883/71864a24-d68e-400b-82da-33cd687dd141" width=200 />
What do you think?
I hope I'm writing this in the correct place, if not please tell me.

**Comment by naveensingh** [comment]
1. I think the previous behavior was confusing. Symbols like  ❌ are usually used to indicate "Close" or "Delete" but it was being used to indicate "Not completed" or "Click to complete" depending on how you see it.

    I might replace the checkbox with
     - ○ : indicates 'Not completed'
     - ✔ : indicates 'Completed'

2. The checkmark/checkbox should always come before the text because 'it is the way':

    <img src="https://github.com/FossifyOrg/Notes/assets/36371707/77f20f9b-f1d5-46f5-bcb3-e846f6175b30" width=264/> <img src="https://github.com/FossifyOrg/Notes/assets/36371707/c8dde88e-7fb8-432c-ad59-0b84bdd148e1" width=344/>

**Reply by Shadow-of-eagle** [reply]
Ok, that makes sense. I think one reason is just that I'm used to it. I still prefer the old design, but it an app should be developed to make sense to the majority of people. Also a checkbox makes sense for a checklist...
